src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -55,6 +55,8 @@ protected internal ContractState CreateContract(byte[] script, byte[] manifest)
 
         protected internal void UpdateContract(byte[] script, byte[] manifest)
         {
+            if (script is null && manifest is null) throw new ArgumentException();
```

Comment:
Not only null, we should also consider situation when length = 0

Reply:
The `length` cannot be `0` in `UpdateContract()`.

https://github.com/neo-project/neo/blob/4066a4f3986a71612dcdeca33a4ecf8934511247/src/neo/SmartContract/ApplicationEngine.Contract.cs#L65-L66

https://github.com/neo-project/neo/blob/4066a4f3986a71612dcdeca33a4ecf8934511247/src/neo/SmartContract/ApplicationEngine.Contract.cs#L82-L83

Reply:
Maybe we can put the logic into a single line

Reply:
No need.